README.md

```diff
@@ -14,7 +14,9 @@
 ## Mission 🚀
 To provide 100 datatable exercises over different sections structured as a course or tutorials to teach and learn for beginners, intermediates as well as experts.
 
-Python's [datatable](https://github.com/h2oai/datatable) package that is synonymous with R's [data.table](https://github.com/Rdatatable/data.table) library can be used for various kinds of data processing, feature engineering and modelling tasks as part of data science pipelines.
+## Datatable 
```

Comment:
Can you suggest an emoji that can be used here?

Reply:
I initially thought of adding the official logo in front of the name, as follows:
![Screenshot 2021-07-05 at 12 56 37 PM](https://user-images.githubusercontent.com/16894695/124433564-83ba8400-dd90-11eb-8362-f4533397ade6.png)

The link to the logo is: https://datatable.readthedocs.io/en/latest/_static/datatable_small_logo.svg

Reply:
Cool, lets see.

Reply:
Done. Added.